data/human/human missions.txt

```diff
@@ -4759,3 +4759,200 @@ mission "Dangerous Games 2"
+			`	You head back to your ship, leaving behind a father survived by his daughter; no more and no less than nature intended.`
+				decline
+			label away
+			`	Riley relaxes slightly once she sees your gun put back in its holster. After the shock has subsided, she finally speaks. "Leave us alone," she says, her voice still trembling. Realizing your time here is up, you exit the hospital and make your way to the spaceport, hoping that that scene did not attract unwanted notice.`
+			choice
+				`	(Turn the Core in for a bounty.)`
+					goto bounty
+				`	(Go back to your ship.)`
+			`	Deciding to leave this planet behind you, you start up your ship and make your way back to the stars.`
+					decline
+			label suggest
+			`	"It's a nice idea," she smiles, "but I'm not a starfarer. The big city is where I'll end back up, eventually."`
+			label goodbye
+			`	Respecting Riley's request to stay with her father, you make your farewells and head for the door.`
+			`	You can feel her smile on your back until the door closes behind you. Despite her happiness, you wonder if her attachment to the AI is healthy.`
+				decline
+			label death
+			`	Riley is stunned for a moment. "You don't know that," she whispers. "Sure, he's a lot more forgetful, and doesn't seem as emotional, but... maybe it just takes some time for the system to warm up. Besides, you don't know what he means to me. What he's done for me. Even if this isn't really my father, I can't turn my back on him."`
+			choice
+				`	(Say goodbye.)`
+				`	(Turn the Core in for a bounty.)`
+					goto bounty
+				`	(Draw your gun and destroy the Core).`
+					goto destroy
+			`	Respecting Riley's request to stay with her father, you make your farewells and head for the door.`
+			`	Despite her previous happiness, you notice that her disposition has become more muted after what you said. You wonder if she has begun to doubt whether her attachment to the AI is healthy.`
+				decline
+			label bounty
+			action
+				payment 100000
+			`	You say your goodbyes and leave. From across the street, you make a call to the local Deep Security office and report what you saw. A slick voice on the other side of your commlink responds.`
+			`	"Ah. Mr. Church has been on our watchlist for some time. This is not exactly surprising. However, we've been unable to get decisive proof on any illegal activity. This information, assuming it is correct, will be a great boon to us. Thank you for the tip." A few minutes pass as you watch their forces descend on the house.`
+			`	Your bank account notifies you that 100,000 credits have been added to your account. Turning away, you head back to your ship.`
+				decline
+			label end
+			`	You say your goodbyes, leaving to your ship and returning to the stars.`
+				decline
+
+
+
+mission "Property Damage"
```

Comment:
I don't really feel like this mission fits in. It just feels like mystery for the sake of mystery ("Why is the AI illegal?" "Because." "Why am I not going on trial?" "Because." "Why are you even telling me all this?" "Because.").

I think my problem with shooting the core stems more from how calm the player character is in spite of the circumstances. Realistically, they would be concerned with getting out as quickly as possible before someone calls the police.